radio/src/pulses/afhds3.cpp

```diff
@@ -145,15 +176,29 @@ enum DeviceAddress {
   FRM303 = 0x04,
   IRM301 = 0x05,
 };
-
-PACK(struct ModuleVersion {
-  uint32_t productNumber;
-  uint32_t hardwereVersion;
-  uint32_t bootloaderVersion;
-  uint32_t firmwareVersion;
-  uint32_t rfVersion;
+PACK(struct ModuleVersion
+{
+  uint16_t company_number;
+  uint32_t tx_id;
+  uint32_t rx_id;
+  uint32_t product_number;
+  uint32_t hardwere_version;
+  uint32_t bootloader_version;
+  uint32_t firmware_version;
+  uint32_t rf_version;
```

Comment:
Either change this back to camel case, or update ln 615-615 with the new names. hardwere_version has a typo btw.